HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -18,6 +17,16 @@
 #                     pset.minGoodStripCharge = cms.PSet(refToPSet_ = cms.string('HLTSiStripClusterChargeCutNone'))
 #     return process
 
+# Matching ECAL selective readout in particle flow, need a new input with online Selective Readout Flags
+def customiseFor17794(process):
+     for edproducer in process._Process__producers.values():
+         if hasattr(edproducer,'producers'):
```

Comment:
Please loop over the module class type, lots of edproducers could have a parameter called producers.